runtime/src/logger.rs

```diff
@@ -35,8 +35,16 @@ where
 
         let item = {
             let metadata = attrs.metadata();
-            let mut visitor = JsonVisitor::default();
 
+            let level = LogLevel::from(metadata.level()) as i32;
+
+            // Ignore span logs from the default level for #[instrument] (INFO) and below.
+            // TODO: make this configurable
+            if level < LogLevel::Warn as i32 {
+                return;
+            }
```

Comment:
I think we only need the `as i32` conversion when it goes into a `LogItem`. You should also not need to convert it to a `LogLevel` yet - aka delay the conversion :smile: